acceptance/testdata/pack_previous_fixtures_overrides/inspect_X.Y.Z_builder_output.txt

```diff
@@ -0,0 +1 @@
+Files like these represent the expected output of calling `pack inspect-builder` on a builder created with pack vX.Y.Z
```

Comment:
Added this example because typically we'd want to use versioned overrides with this directory. I think that was the root of the latest problem (no version).